src/test/java/seedu/address/model/person/SubjectTest.java

```diff
@@ -62,4 +63,19 @@ public void isValidSubjectsByLevel_invalidLevel_failure() {
         assertFalse(Subject.isValidSubjectsByLevel(new Level("NONE NONE"), SUBJECT_ARRAY));
     }
 
+    @Test
+    public void subject_case_insensitive() {
+        assertTrue(new Subject("MATH").equals(new Subject("math")));
+    }
+
+    @Test
+    public void toStringTest() {
+        Subject s = new Subject("MATH");
+        Subject s2 = new Subject("math");
+        Subject s3 = new Subject("mAtH");
+        String expected = "[MATH]";
+        assertEquals(s.toString(), expected);
+        assertEquals(s2.toString(), expected);
+        assertEquals(s3.toString(), expected);
```

Comment:
same as LevelTest, should tests be added under isValidSubject to check that it is true for lower case, with extra space, etc?